src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -204,6 +204,15 @@ public void testEnumsAndAnnotations() throws Exception {
         verify(checkConfig, getPath("InputWriteTag2.java"), expected);
     }
 
+    /**
+     * A custom verify method.
+     * We suppress Resource close as This Check is deprecated
+     * @param checker {@link Checker} instance.
+     * @param processedFiles list of files to verify.
+     * @param messageFileName message file name.
+     * @param expected an array of expected messages.
+     * @throws Exception might happen
+     */
```

Comment:
Why is this documentation needed when no code was changed? This is a test area.

Reply:
yes, looks strange.